src/Billing/Controllers/StripeController.cs

```diff
@@ -750,7 +750,8 @@ private async Task<bool> AttemptToPayInvoiceWithAppleReceiptAsync(Invoice invoic
             GatewayType.AppStore, receiptLastTransactionId);
         if (existingTransaction != null)
         {
-            _logger.LogWarning("There is already an existing transaction for this Apple receipt.",
```

Comment:
ℹ️ There are a lot interpolated strings in logs today and we'll want to catch them with rules most likely, but `dotnet format` singled out this warning.